src/config.rs

```diff
@@ -305,6 +305,32 @@ impl Config {
         self.write()?;
         Ok(())
     }
+    pub fn keys(&self) -> Vec<&str> {
+        let mut public_keys = vec![
+            "android_branches",
+            "android_manifest_url",
+            "default_cros_checkout",
+            "default_cros_reference",
+            "ssh_overrides",
+            "tast_bundles",
+            "ssh_port_search_timeout",
+            "default_ipv6_prefix",
+        ];
+        let mut internal_keys = vec![
+            "is_internal",
+            "is_internal_auth_valid",
+            "acloudw_cmd_path",
+            "acloudw_config_path",
+            "android_target_for_vm_type",
+            "arc_vm_cheeps_image",
+            "arc_vm_betty_image_for_branch",
+            "arc_container_cheeps_image_for_branch",
+        ];
+        if self.is_internal() {
```

Comment:
I think there is no need to separate internal keys and external ones as all the config keys will appear in the source code anyways...

Reply:
OK, I just thought these are not useful for non-internal users.